### ISSUE TITLE
Error 500, no way to debug anything, try-catch returns error 500 too

### ISSUE THREAD

**Issue by rt-2** [mgp25/Instagram-API#1559]
Hi
```
<?php
	set_time_limit(0);
	date_default_timezone_set('UTC');
	require __DIR__.'../vendor/autoload.php';
	echo 'works';
?>
```
returns error 500.
```
<?php
	set_time_limit(0);
	date_default_timezone_set('UTC');
	require '../vendor/autoload.php';
	echo 'works';
?>
```
works...
```
<?php
	set_time_limit(0);
	date_default_timezone_set('UTC');
	require '../vendor/autoload.php';
	try {
		$debug = true;
		$ig = new \InstagramAPI\Instagram($debug);
		$ig->setUser(IG_USER, IG_PASSWORD);
		$ig->login();
		$result = $ig->comment('14123451234567890_1234567890', "Hello World");
		var_dump($result);
	} catch (\Exception $e) {
		echo $e->getMessage()."\n";
	}
	echo 'works';
?>
```
returns error 500.

I don't know how to test, I was not familiar with Composer, but with some playing around I managed to run `composer install` without any errors and it created the wanted 'vendor' directory with the 'autoload.php'.
But everything I try returns a blank error 500....

Thank you,
rt-2

**Comment by SteveJobzniak** [comment]
Error 500 means "Internal Server Error". Instagram should not respond with that. You may be using a broken proxy?

Or even more likely is that your error 500 actually comes from your own webserver. Check your own PHP code. And try running the script in terminal/command line with php directly instead.

The code you list at the top as "returns error 500" just includes a PHP file and doesn't even create an instance of our library. So yes I think this is a problem with your own webserver.

**Comment by rt-2** [comment]
"Instagram should not respond with that" Interesting, but it does.
Can you explain how I could 'debug my perfectly good (working for 10 year without problem, recently upgraded for composer) web server', I never had any problem and I actually work in the server field.
Can you help me find out what am I missing  on my web server? I installed everything this package asked for.
Thank you,
rt-2

**Comment by rt-2** [comment]
There is actually no "requirements" section in this package weirdly enough,, but is it php7 only? or also compatible on let say 5.6?

**Comment by rt-2** [comment]
I guess you stopped reading my issue after the first example, so I posted another to corner even more your module as the culprit.
https://github.com/mgp25/Instagram-API/issues/1561
By the way this is not a test server this is a web server in production hosting 10+ web applications for clients.
I installed everything you asked in the readme and it still doesn't work, your plugin IS the problem, or your lack or explanation maybe.
An error 500 means internal server error yes, but it doesn't mean it's the server itself,,, your script is internal to my server when it is executed BY THE WAY.
SMH,
rt-2
